src/tile-board/Tileboard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Tile from './Tile';
+
+function TileBoard() {
+  const arrayGrid = createArrayGrid();
+  
+  return (
+    <div className='container text-center mt-5'>
+      {
+        // Maps over the array grid, adding a <div> element that wraps around each row.
+        arrayGrid.map((row, key) => {
+          return <div key={key}>{row}</div>;
```

Comment:
Using an array index as a key is a "last resort" ([docs](https://reactjs.org/docs/lists-and-keys.html#keys)), but makes sense in this case. The docs go into detail but the main thing to watch out for is if the contents of the array change but the key stays the same things can get weird (which I don't think applies in your specific case here).